qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -22,6 +22,8 @@
 from qiita_core.exceptions import IncompetentQiitaDeveloperError
 import qiita_db as qdb
 
+from qiita_db.metadata_template.constants import SAMPLE_TEMPLATE_COLUMNS
```

Comment:
IMOO kind of odd that we are importing all `qiita_db as qdb` and then importing this. If you want to avoid some long strings what about in the `setUp` so something like `self.STC = qdb.metadata_template.constants.SAMPLE_TEMPLATE_COLUMNS` and then just call `self.STC`?

Reply:
Agreed, and it yes, it was to avoid some really long identifier names. How about using this top-level alias, instead of making it a class variable?

STC = qdb.metadata_template.constants.SAMPLE_TEMPLATE_COLUMNS

Reply:
will that make it a global variable? I always _feel funny_ about them ...

Reply:
I believe in Python, it is global only to the module, and we can prevent it from being imported with the others by prefixing with '__'. The standard way to create a global variable in Python is to declare the variable in a module, and import that module into every file.